lib/presentation/home/screens/message_with_retry.dart

```diff
@@ -9,10 +9,12 @@ class MessageWithRetryScreen extends StatelessWidget {
     super.key,
     required this.message,
     required this.onTapRetry,
+    this.onTapChangePath,
```

Comment:
I'm against that we should extend MessageWithRetry widget for that case. It would be great to have a separate widget for that state. 
You can copy-paste `message_with_retry.dart` and name it like `LolNotLaunchedOrWrongPathProvidedScreen`